src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -307,14 +309,21 @@ protected function createContainer(array $data = array())
 
     protected function createContainerFromFile($file, $data = array())
     {
-        $container = $this->createContainer($data);
-        $container->registerExtension(new FrameworkExtension());
-        $this->loadFromFile($container, $file);
+        $dataHash = md5(serialize($data));
```

Comment:
Sorry for the nitpicking: should we rename `$dataHash` to `$cacheKey` or `$cacheId` ?